date-lib.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const modules = [
+  'converter',
+  'format',
+  'from-now',
+].map(path => require('./lib/' + path));
+
+module.exports = Object.assign({}, ...modules);
```

Comment:
Nit: I'd not recommend using this pattern, while it looks nice but it makes it very hard for IntelliSense (i.e. code completion etc) to understand and process code.